Source/Core/DolphinQt2/Settings/WiiPane.cpp

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Settings/WiiPane.h"
+
+#include <QCheckBox>
+#include <QGroupBox>
+#include <QLabel>
+#include <QSlider>
+#include <QVBoxLayout>
+#include <QWidget>
+
+#include "Core/ConfigManager.h"
+
+WiiPane::WiiPane(QWidget* parent) :QWidget(parent)
+{
+  CreateLayout();
+  ConnectLayout();
+
+  LoadConfig();
+}
+
+void WiiPane::CreateLayout()
+{
+  m_main_layout = new QVBoxLayout;
+  CreateMisc();
+  m_main_layout->setContentsMargins(0, 0, 0, 0);
+  m_main_layout->addStretch(1);
+  setLayout(m_main_layout);
+}
+
+void WiiPane::ConnectLayout()
+{
+  connect(m_wiimote_speaker_volume, &QSlider::valueChanged, this, &WiiPane::OnSaveConfig);
+  connect(m_wiimote_motor, &QCheckBox::toggled, this, &WiiPane::OnSaveConfig);
+}
+
+void WiiPane::CreateMisc()
+{
+  auto* basic_group = new QGroupBox(tr("Misc Settings"));
+  auto* basic_group_layout = new QVBoxLayout;
+  basic_group->setLayout(basic_group_layout);
+  m_main_layout->addWidget(basic_group);
+  m_wiimote_motor = new QCheckBox(tr("Enable Wiimote Motor"));
+  // Speaker Volume Slider
+  m_wiimote_speaker_volume_label = new QLabel(tr("Wiimote Speaker Volume"));
+  m_wiimote_speaker_volume = new QSlider(Qt::Horizontal);
+  m_wiimote_speaker_volume->setFocusPolicy(Qt::StrongFocus);
+  m_wiimote_speaker_volume->setTickPosition(QSlider::TicksBothSides);
+  m_wiimote_speaker_volume->setTickInterval(8);
```

Comment:
Slider ticks aren't used anywhere else in DolphinQt2, or in the original DolphinWX UI. If the slider value needs to be communicated more accurately than eyeballing its position between "Min" and "Max" (which, yeah, it probably should be, @MayImilae ?), the pattern used elsewhere is to attach a dynamic label afterwards. The audio stretching buffer size slider is a good example of this pattern:

<img width="286" alt="screen shot 2017-07-21 at 3 14 36 pm" src="https://user-images.githubusercontent.com/594093/28484460-526a11d6-6e27-11e7-8bc6-5e53745c7aa4.png">